config/environments/development.rb

```diff
@@ -14,7 +14,7 @@
   config.public_file_server.enabled        = false
 
   # Don't care if the mailer can't send.
-  config.action_mailer.raise_delivery_errors = false
+  config.action_mailer.raise_delivery_errors = true
```

Comment:
Mysterious! Usually actionmailer emails log the full content to the rails console in dev, but if you give me steps to recreate I can look into this.